packages/IMAPClient-UI.package/ICFolderDialog.class/instance/login..st

```diff
@@ -0,0 +1,8 @@
+action - emails
+login: anAccountInfo
+
+	anAccountInfo password: (Project uiManager requestPassword: 'Please enter your password!').
+		
+	((ICEndPoint new) testAccountWith: anAccountInfo)
```

Comment:
For this, probably the original (commented out) `testAccountWith` should be restored (currently returns constant true).